data/driver.list.in

```diff
@@ -77,10 +77,12 @@
 "APC"	"ups"	"3"	"Back-UPS BF500"	"USB"	"usbhid-ups"
 "APC"	"ups"	"3"	"BACK-UPS XS LCD"	"USB"	"usbhid-ups"
 "APC"	"ups"	"3"	"Back-UPS XS 1000M (Back-UPS Pro 1000, Model BX1000M)"	"USB"	"usbhid-ups"	# https://github.com/networkupstools/nut/issues/139
-"APC"	"ups"	"3"	"SMC2200BI-BR"	"USB"	"usbhid-ups"	# https://github.com/networkupstools/nut/issues/557
-"APC"	"ups"	"3"	"Smart-UPS (USB)"	"USB"	"usbhid-ups"
-"APC"	"ups"	"3"	"Smart-UPS 750 (SMT750I, USB)"	"USB"	"usbhid-ups"
-"APC"	"ups"	"3"	"Smart-UPS X 750 (SMX750I, USB)"	"USB"	"usbhid-ups"
+"APC"	"ups"	"3"	"SMC2200BI-BR"	"USB"	"apc_modbus"	# https://github.com/networkupstools/nut/issues/557
+"APC"	"ups"	"3"	"Smart-UPS (USB)"	"USB"	"apc_modbus"
+"APC"	"ups"	"3"	"Smart-UPS 1500 (SMT1500I, USB)"	"USB"	"apc_modbus"
+"APC"	"ups"	"3"	"Smart-UPS 750 (SMT750I, USB)"	"USB"	"apc_modbus"
+"APC"	"ups"	"3"	"Smart-UPS X 750 (SMX750I, USB)"	"USB"	"apc_modbus"
+"APC"	"ups"	"3"	"Smart-UPS X 1500 (SMX1500I, USB)"	"USB"	"apc_modbus"
```

Comment:
Any serial or TCP models to report as successes here? (NUT DDL PRs with data dumps, or at least comment posts here, from various devices, would also be appreciated).

Reply:
@jimklimov @PrplHaz4 tested his SMX1500I using TCP and I tested my SMT1500I using serial.

Reply:
Also, regarding this block as a removal: didn't `usbhid-ups` at least somehow handle those devices? It is okay to list several suitable drivers (on separate lines) - e.g. `nutdrv_qx` and `blazer_(ser|usb)` have a lot of that ;)

Reply:
Yes, all of these models should work with `usbhid-ups` too, I'll add that.